cifar10/main.py

```diff
@@ -162,7 +165,7 @@ def run(
         resume_from (str, optional): path to checkpoint to use to resume the training from. Default, None.
         with_pbar(bool): if True adds a progress bar on training iterations.
         with_amp(bool): if True uses torch native AMP
-        rescaled_size (int, optional): if provided then input image will be rescaled to that value.
+        rescale_size (int, optional): if provided then input image will be rescaled to that value.
```

Comment:
It would be good to add `cutmix_beta` and `cutmix_prob` here as well.

Reply:
Aww true !